src/HasteModuleLoader/HasteModuleLoader.js

```diff
@@ -204,10 +204,8 @@ class Loader {
 
     // Use this name for the module wrapper for consistency with node.
     const evalResultVariable = 'Object.<anonymous>';
```

Comment:
I think this name isn't useful anymore, right? Now it will print `Object.eval` instead of `Object.<anonymous>`.